ctest-next/src/ast/constant.rs

```diff
@@ -0,0 +1,41 @@
+/// Represents a constant variable defined in Rust.
+pub struct Const {
+    public: bool,
+    ident: syn::Ident,
```

Comment:
I'm changing my mind a bit here; I think identifiers should be stored as strings rather than `Ident` after all, so the `.ident()` method doesn't need to keep cloning it.

Add a `type BoxStr = Box<str>;` to `lib.rs` and use that for the `ident` fields, you can call `.to_string().into_boxed_str()` from `Ident`.